data/boarding missions.txt

```diff
@@ -61,6 +61,160 @@ mission "Assisting Merchant (Large)"
 
 
 
+mission "Asteroid Course A"
+	assisting
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	destination "Earth"
+	on offer
+		conversation
+			`As you are approaching the <origin>, your radar sensors detect that the ship you are assisting is in the middle of a large asteroid's course! While in a normal flight this wouldn't cause much trouble, a ship without active shields and that has already suffered heavy damages could be completely destroyed.`
+			branch unarmed
+				"armament deterrence" == 0
+			choice
+				`	(Shoot towards the asteroid to destroy it.)`
```

Comment:
"towards" is superfluous here. I'd drop "to destroy it" because we can't guarantee that are guns will actually do the job.
```suggestion
				`	(Shoot the asteroid.)`
```